modules/ROOT/pages/logs-us-eu.adoc

```diff
@@ -199,6 +213,26 @@ This example shows an expanded log entry in JSON format.
 }
 ----
 
+[[search-agentic]]
+== Get Agent, Broker, and MCP Server Logs
+
+API Manager and Agent Visualizer provide links to logs for individual agents, brokers, and MCP servers. To search for these logs from Anypoint Monitoring, provide the name of the instance in the search field with any filters to narrow your query.
+
+[IMPORTANT]
```

Comment:
@marianogonzalez do you see issues with the content about enabling insecure logging or whatever else here?

Reply:
thx, @marianogonzalez I incorporated your change. 😄